Lombiq.HelpfulLibraries.OrchardCore/Mvc/ShapeResultExtensions.cs

```diff
@@ -18,7 +18,7 @@ public static class ShapeResultExtensions
         string name,
         double priority,
         string placement = CommonLocationNames.Parts) =>
-        shapeResult.Location(FormattableString.Invariant($"{placement}#{name}:{priority}"));
+        shapeResult.Location(string.Create(CultureInfo.InvariantCulture, $"{placement}#{name}:{priority}"));
```

Comment:
Why did you change these?

Reply:
With the newest Meziantou.Analyzer version, using FormattableString gives you an [analyzer warning](https://github.com/meziantou/Meziantou.Analyzer/blob/main/docs/Rules/MA0111.md).

Reply:
Please figure out what there is this rule. There's no explanation and it doesn't make sense to me, it's not simpler to write or easier to read.

Reply:
So I got [this](https://github.com/meziantou/Meziantou.Analyzer/issues/359#issuecomment-1218017333) reply to my question, what's your opinion, should we stick to FormattableString and suppress this warning or go with string.Create?

Reply:
I see, In our use-cases it seems more like a micro-optimization. Please check out if you can add a static utility method or an extension method to have a shortcut for this; if not (formattable strings are special) then let's revert and disable the rule, since this is not a performance issue for us currently (and if there's something obvious for optimization I'd expect the .NET team to address it).

Reply:
Yep, you were right, as I have found out, you can't really write an extension method that we would want here, so I will revert the changes and disable the rule.

Reply:
Note that I also mentioned a utility method. That doesn't work either?

Reply:
Same case with that as well.

Reply:
Let's revert his and disable the rule, then.